pyproject.toml

```diff
@@ -669,7 +669,7 @@ filterwarnings = [
 ]
 
 [tool.ruff]
-required-version = ">=0.4.7"
+required-version = ">=0.4.8"
```

Comment:
I had a chat with @cdce8p and we both had the feeling that this check is more annoying than helpful at the moment. For example in a change like this, if we didn't have any (formatting) issues, I don't expect us to introduce them now.

And after all, the CI runs the latest, so it's only a matter of 2 minutes to know if there is a mistake

Reply:
Tbh I thought it was an issue, it certainly was one in the past. I don't usually install the latest test dependencies every day so for a time I regularly saw something like this
```
ruff failed
  Cause: Required version `==0.4.7` does not match the running version `0.4.8`
```

However, I can't reproduce it now. It might have been that at some point this was an exact pin which caused it but who knows 🤷🏻‍♂️

--
Anyway I'm not sure this is all too relevant. Pre-commit uses the version defined in `.pre-commit-config.yaml`. So it's probably only for those of us who run `ruff` directly, but who continues to do that?

Maybe an idea could be to only update the version here for `minor`, not `patch`, releases. I.e. change that to `>=0.4`.

Reply:
I had this regularly - until we fixed the container in #113517